### ISSUE TITLE
undefined is not a function(near .....reactDevTools.connectToDevTools. ) issue

### ISSUE THREAD

**Issue by shravanteegala** [facebook/react-devtools#1266]
![screen shot 2019-01-04 at 2 19 06 pm](https://user-images.githubusercontent.com/13646977/50679978-c36f4c00-102b-11e9-94f3-0d19716ce95c.png)

**Comment by gokulnathperiasamy** [comment]
Any alternate way or workaround ?

**Comment by shravanteegala** [comment]
i fix using below command.
npm install --save-dev react-devtools-core@3.4.2 Resolved the issue

**Comment by herodrigues** [comment]
This was broken in latest release `v3.5.0`. Roll back to `v3.4.3` and everything should be fine.

**Comment by bvaughn** [comment]
Hi all 👋 Sorry for the trouble 😦

I believe this was something I broke when upgrading the React DevTools project's Babel and Webpack versions in https://github.com/facebook/react-devtools/pull/1235, and we're just now encountering it because we haven't released the `react-devtools-core` package since then.

I just published `react-devtools-core` v3.5.1. Can anyone let me know if this resolves the issue? (I'm unable to repro the original reported issue so it's hard for me to verify the fix.)
